ops/infrastructure/webapp_playbook.yml

```diff
@@ -85,6 +85,54 @@
         path: /tmp/aws-efs-utils.rpm
         state: absent
 
+- name: Copy script to renew TLS certificates
+  hosts: name_gigadb_webserver_{{gigadb_env}}*
+  tags:
+    - setup-tls-certs
+
+  tasks:
+    - name: Create secrets file
+      ansible.builtin.template:
+        src: "../../../../ops/configuration/tls-certs-conf/tls-certs-secrets.j2"
+        dest: /home/ec2-user/.tls-certs-secrets
+        owner: ec2-user
+        group: ec2-user
+        mode: '0644'
+
+    - name: Copy script
+      ansible.builtin.copy:
+        src: "../../../../ops/scripts/renew_cert.sh"
+        dest: /usr/local/bin/renew_cert
+        owner: ec2-user
+        group: ec2-user
+        mode: a+x
+
+    - name: Create directory for renew detection
+      ansible.builtin.file:
+        path: "/home/ec2-user/cert_renew_detection"
+        state: directory
+        owner: ec2-user
+        group: ec2-user
+        mode: g-rw,o-rw
+
+    - name: Create log file
+      ansible.builtin.file:
+        path: /var/log/renew_certs.log
+        state: touch
+        group: root
+        owner: ec2-user
+        mode: g-rw,o-rw
+
+    - name: Create cron job to renew TLS certificates on every Sunday at midnight
+      ansible.builtin.cron:
+        name: "Renew TLS certificates on every Sunday at midnight"
+        weekday: "0"
+        minute: "0"
+        hour: "0"
+        user: "ec2-user"
+        job: "/usr/local/bin/renew_certs > /var/log/renew_certs.log 2>&1"
```

Comment:
Is `/usr/local/bin/renew_certs` correct? Or should it be `/usr/local/bin/renew_cert` - see line 105?

Reply:
Fixed in 1ca59316b